docs/cli/build.md

```diff
@@ -133,6 +133,58 @@ RUN --mount=type=secret,id=netrc,mode=0666,dst=/home/app/.netrc \
 
 Bear in mind that at this time, `GITHUB_TOKEN` in a GitHub Action cannot be used to clone other repositories, even within the same organisation.
 
+### Private npm modules
+
+Get the OpenFaaS Pro plugin and enable it:
+
+```bash
+faas-cli plugin get pro
+faas-cli pro enable
+```
+
+Create a function:
+
+```bash
+export OPENFAAS_PREFIX=openfaasltd
+faas-cli template store pull node17
+faas-cli new --lang python3-http withprivatenpm
+mv withprivatenpm.yml stack.yml
+```
+
+Edit your `~/.npmrc` file to include the authentication token for your npm registry.
```

Comment:
Isn't there an npm login command for this?

What about a section with the instructions you used for GHCR?

Reply:
I linked to the instructions for both the GithHub Packages registry and the npmjs registry.